src/features/search/components/FilterPageButtons/FilterPageButtons.tsx

```diff
@@ -13,19 +13,27 @@ type Props = {
   onSearchPress: () => void
   isSearchDisabled?: boolean
   children?: never
+  willTriggerSearch?: boolean
```

Comment:
si je comprends bien `willTriggerSearch` permet juste de changer le wording et savoir si ça cherche directement ou non

je n'aime pas les `boolean` : mes expériences passées m'ont souvent montrées des difficultés de lors de la maintenance de `boolean` et / ou des concepts éloignés du langages métiers

que penseriez-vous d'un truc du genre (je vous laisse voir pour le nommage) ?

```ts
enum FilterBehavior {
  Search,
  ApplyWithoutSearching
}
```

```suggestion
  filterBehavior: FilterBehavior
```

Reply:
L'idée est aussi bonne mais avoir un `enum` de 2 valeurs juste pour remplacer `true/false` je sais pas si ça n'ajoute pas de la complexité pour pas grand chose... D'autant que l'enum n'aura jamais d'autre valeur.

Reply:
je ne vois aucun soucis à avoir un `enum` avec 2 valeurs

si ça simplifie la lecture et que c'est plus explicite sur le comportement, je préfère

Reply:
On a fait les modifications nécessaires